pycbc/conversions.py

```diff
@@ -395,6 +425,107 @@ def secondary_spin(mass1, mass2, spin1, spin2):
     ss[mask] = spin1[mask]
     return _formatreturn(ss)
 
+def primary_xi(spinx, spiny):
+    """Returns the effective precession spin argument for the larger mass.
+    """
+    return chi_perp_from_spin1x_spin1y(spinx, spiny)
+
+def secondary_xi(mass1, mass2, spinx, spiny):
+    """Returns the effective precession spin argument for the smaller mass.
+    """
+    q = q_from_mass1_mass2(mass1, mass2)
+    a1 = 2 + 3 * q / 2
+    a2 = 2 + 3 / (2 * q)
+    return a1 / (q**2 * a2) * chi_perp_from_spin1x_spin1y(spinx, spiny)
```

Comment:
Same comment as above... rename this `xi2_from_mass1_mass2_spinx_spiny`, and add a new function called `secondary_xi` that takes mass1, mass2, spin1x, spin1y, spin2x, spin2y, pulls out the secondary spins, and passes that to the former function.